src/test/scala/com/microsoft/ml/spark/core/test/base/SparkSessionFactory.scala

```diff
@@ -47,7 +47,7 @@ object SparkSessionFactory {
     val sess = SparkSession.builder()
       .config(conf)
       .getOrCreate()
-    sess.sparkContext.setLogLevel(logLevel)
+    sess.sparkContext.setLogLevel("INFO")
```

Comment:
done on purpose?

Reply:
Yep, otherwise during tests at some point the root log level will be set as warning and I can't see info-level logs.

Reply:
I dont think this will affect the deployed Jar so I would suggest keeping it at the logLevel for now. This only affects tests in the build